client/interfaces.go

```diff
@@ -286,6 +286,9 @@ type InstanceServer interface {
 	RenameNetworkACL(name string, acl api.NetworkACLPost) (err error)
 	DeleteNetworkACL(name string) (err error)
 
+	// Network allocations functions ("network_allocations" API extension)
+	GetNetworkAllocations(allProjects bool) (allocations []api.NetworkAllocations, err error)
```

Comment:
Please can you move the commit that defines the api.NetworkAllocations before this one in the PR.